app/src/main/res/layout/item_susi_map.xml

```diff
@@ -30,7 +30,7 @@
 
                 <TextView
                     android:id="@+id/text"
-                    android:layout_width="wrap_content"
+                    android:layout_width="240dp"
```

Comment:
Yeah, that's about gravity of the text. Changing it to constant is not the best way to solve this. Try setting `layout_gravity` on the container layout or `gravity` on the text itself